.github/workflows/ci.yaml

```diff
@@ -60,6 +60,74 @@ jobs:
           name: gradle_spotbugs_reports
           path: /tmp/gradle_spotbugs_reports
 
+  docker:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Dockerfile Lint for Scalar DB Server
```

Comment:
If this action fails because a vulnerability is found, I think the following actions will be skipped. 
Should the following actions be configured to always run using the `if: always()` syntax?

Reply:
Added the `if: always()` syntax. Thanks.